cli-plugins/apps/how-to-examples.hbs.md

```diff
@@ -1,1119 +0,0 @@
-# Workload Examples
-
-Use the following flags with the Apps CLI plug-in.
-
-## <a id='custom-registry'></a> Custom registry credentials
```

Comment:
cli-plugins\apps\command-reference\workload_create_update_apply.hbs.md
line 957 links to this section
What should it link to now?

Reply:
This file is moved in another PR. I'll revisit it and fix the link